Dev-Corner/slides/toolshed.html

```diff
@@ -0,0 +1,801 @@
+                            - Click on the workflow at the bottom of the “Contents of this repository” section
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ###  How to install a workflow?
+                            - Repository Actions > Import workflow to Galaxy
+                            <img src="../images/exportWorkflow_ss.png" class="no-border" >
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ###  How to install a workflow?
+                            <img src="../images/workflowOK_ss.png" class="no-border" >
+                            - Done!
+                        </script>
+                    </section>
+                </section>
+                <section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ## 3. Share your tool on a Tool Shed
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### 3.1 Tool with dependencies
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ###  Tool with dependencies
+                            <img src="../images/toolshedPackage.png" class="no-border" >
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ###  Tool with dependencies
+                            - Report the requirement in your wrapper in order to **load environment with Conda**
+                            `tandem_repeats_finder_wrapper.xml`
+                            ```
+                            <requirements>
+                              <requirement type="package" version="4.0">tandem_repeats_finder</requirement>
+                            </requirements>
+                            ```
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### 3.2 Tool suite: relationship between required repositories
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ###  Tool suite
+                            <img src="../images/toolsuite.png" class="no-border" >
```

Comment:
@slegras 
We don't understand in the picture that this method is the wrong without the oral speech